test/SlowTests/Issues/RavenDB-14541.cs

```diff
@@ -0,0 +1,126 @@
+﻿
+
+using System;
+using System.Linq;
+using Confluent.Kafka;
+using FastTests;
+using Raven.Client.Documents;
+using Raven.Client.Documents.Linq;
+using Raven.Client.Documents.Queries;
+using SlowTests.Core.Utils.Entities;
+using Tests.Infrastructure.Entities;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues;
+
+public class RavenDB_14541: RavenTestBase
+{
+    public RavenDB_14541(ITestOutputHelper output) : base(output)
+    {
+    }
+    private class Address
+    {
+        public string CountryState { get; set; }
+        public string City { get; set; }
+    }
+
+    private class State
+    {
+        public string Name { get; set; }
+    }
+
+    private class QueryResult
+    {
+        public string Comapny { get; set; }
+    }
+
+    [Fact]
+
+    public void ShouldThrowNotSupportedException()
+    {
+        try
+        {
+            using (DocumentStore store = GetDocumentStore())
+            {
+                using (var session = store.OpenSession())
+                {
+                    session.Store(new Order { Company = "1", Employee = "employees/1" }, "orders/1-A");
+                    session.Store(new Order { Company = "2", Employee = "employees/2" }, "orders/2-A");
+
+
+                    session.Store(new Employee { FirstName = "a" }, "employees/1");
+                    session.Store(new Employee { FirstName = "b" }, "employees/2");
+
+                    session.SaveChanges();
+                }
+
+                using (var session = store.OpenSession())
+                {
+
+                    var query3 = from u in session.Query<Order>()
+                        let includes = RavenQuery.Include<Order>(u.Employee)
+                        select new QueryResult
+                        {
+                            Comapny = u.Company
+                        };
+                }
+            }
+        }
+        catch (Exception e)
+        {
+            Assert.NotNull(e);
+            Assert.Contains("You can't use the include that way try: let _= RavenQuery.Include<T>()", e.InnerException.Message);
```

Comment:
Why is this an error?

Reply:
The error is using a wrong name in the let when using 'RavenQuery.Include'.

The expected query is supposed to be as follows:
`var query = from u in session.Query<Order>()
 let _ = RavenQuery.Include<Order>(u.Employee) 
select new QueryResult { Comapny = u.Company };`

Reply:
we don't want the user to assign the included value to a variable, since it is included to session. So we allow only to use `let _+`

Reply:
Why are we bothering with that at all?
If they try to assign the variable, they will get no results, right?
An array of `null` ? 

We can make this return the actual document ids, by the way, and that would be far cleaner.